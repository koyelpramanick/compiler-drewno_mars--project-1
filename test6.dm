"backslash followed by space: \ is not allowed"
"\q"
"\\""
"unterminated
"test"
"\q \end"
"\n \q"
"testing // inside string literal"
"blah blah \" blah"
"'quote' in a string"
"@@@$$#$$##@@!!@#$$%%^&*()))(**&&^^%$$##@"
"unterminated string literal with \bad escape sequence
"bad escaped character: \a AND not terminated