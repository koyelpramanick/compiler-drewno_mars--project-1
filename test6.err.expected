FATAL [1,1]-[1,48]: String literal with bad escape sequence detected
FATAL [2,1]-[2,5]: String literal with bad escape sequence detected
FATAL [3,5]-[3,6]: Unterminated string literal detected
FATAL [4,1]-[4,14]: Unterminated string literal detected
FATAL [6,1]-[6,10]: String literal with bad escape sequence detected
FATAL [7,1]-[7,8]: String literal with bad escape sequence detected
FATAL [12,1]-[12,55]: Unterminated string literal with bad escape sequence detected
FATAL [13,1]-[13,46]: Unterminated string literal with bad escape sequence detected
